assets/css/styles.css

```diff
@@ -1,5 +1,165 @@
-/* Edo Font */
```

Comment:
Don't remove the edo font bcs it will affect for the main heading's font style @Mayureshkaulwar

Reply:
Can you please explain me in detail which change I have to do.

Reply:
Don't remove the edo font type from styles.css file @Mayureshkaulwar

Reply:
please merge pull request, i have added edo font